web/admin/src/Admin.js

```diff
@@ -220,6 +262,71 @@ export default class Admin extends PureComponent {
     )
   }
 
+  formatResponsesForExport = () => {
+    const { sessionId, users } = this.state
+    this.props.fbc.database.private.adminableUsersRef().once('value', data => {
+      const answersPerUser = data.val() || {}
+      // if (answersPerUser[id].responses) {
+      const answers = Object.keys(answersPerUser)
+        .filter(id => {
+          if (answersPerUser[id].responses) return answersPerUser[id].responses[sessionId] != null
```

Comment:
This filter function doesn't always return something.  Maybe:

.filter(id => answersPerUser[id].responses && answersPerUser[id].responses[sessionId] != null)